README.md

```diff
@@ -355,15 +358,12 @@ ChatView(
 )
 ```
 
-14. Get imagepath from image picker and add imagepicker icon configuration.
+14. Add image picker icon configuration.
 ```dart
 ChatView(
   ...
   sendMessageConfig: SendMessageConfiguration(
     imagePickerIconsConfig: ImagePickerIconsConfiguration(
-      onImageSelected: (imagePath, error){
```

Comment:
any specific reason for removing this ?

Reply:
As we have this call back in onSendTap callback. That's the reason

Reply:
Also, it has been removed from code as well